services-directions/src/test/java/com/mapbox/api/directions/v5/models/RouteOptionsTest.java

```diff
@@ -190,4 +190,28 @@ public void directionsRequestResult_doesContainOverview() throws Exception {
 
     assertEquals(DirectionsCriteria.OVERVIEW_SIMPLIFIED, route.routeOptions().overview());
   }
+
+  @Test
+  public void toJson_fromJson() throws Exception {
```

Comment:
Normally is good to structure the tests following the triple A 👀 https://speakerdeck.com/guardiola31337/elegant-unit-testing-mobilization-2016?slide=40 http://wiki.c2.com/?ArrangeActAssert
If the `A`rrange part gets too long we should create factory or builder methods.
What about removing some "unnecessary" new lines so that the 3 parts of the test (`A`rrange, `A`ct and `A`ssert) are visually 👌?

Reply:
What about updating the naming here to be more specific / descriptive? The test names should reflect what we're testing so if you 👀 at it you know what's going on at a glance.